src/electionguard-ui/ElectionGuard.UI/ViewModels/CreateMultiTallyViewModel.cs

```diff
@@ -0,0 +1,353 @@
+﻿using CommunityToolkit.Maui.Storage;
+using CommunityToolkit.Mvvm.Input;
+
+namespace ElectionGuard.UI.ViewModels;
+
+[QueryProperty(MultiTallyIdParam, nameof(MultiTallyId))]
+
+public partial class CreateMultiTallyViewModel : BaseViewModel
+{
+    public const string MultiTallyIdParam = "MultiTallyId";
+
+    private TallyService _tallyService;
+    private ElectionService _electionService;
+    private ManifestService _manifestService;
+    private KeyCeremonyService _keyCeremonyService;
+    private BallotUploadService _ballotUploadService;
+    private BallotService _ballotService;
+    private TallyJoinedService _tallyJoinedService;
+    private MultiTallyService _multiTallyService;
+
+    public CreateMultiTallyViewModel(
+        IServiceProvider serviceProvider,
+        TallyService tallyService,
+        ManifestService manifestService,
+        ElectionService electionService,
+        KeyCeremonyService keyCeremonyService,
+        BallotUploadService ballotUploadService,
+        BallotService ballotService,
+        TallyJoinedService tallyJoinedService,
+        MultiTallyService multiTallyService) : base("CreateMultiTally", serviceProvider)

```

Comment:
This is not something that needs to be in this PR.  That is an overall concept change from the original designs.  It is doable but will need to look at how we are using this and how it can be simplified.